src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Disposal.java

```diff
@@ -68,6 +68,9 @@
     @OneToMany(targetEntity = Requirement.class, mappedBy = "disposal")
     private List<Requirement> requirements;
 
+    @OneToOne(mappedBy = "disposal")
```

Comment:
Maybe check if this needs to be lazy, I think we might use Disposal a lot, not 100% sure

Reply:
I did have  a look at this and we can make this lazy but it won't actually bring us any benefit on it's own because this field will be retrieved automatically in all cases regardless because the transformer will always check to see if unpaid work is there. We could add an `includeUnpaidWork` query param to the endpoint and only fetch upw conditionally.

Reply:
Fair enough